template/express-rest-api/.buildkite/pipeline.yml

```diff
@@ -11,8 +11,9 @@ configs:
     - &docker-ecr-cache
       seek-oss/docker-ecr-cache#v2.1.0: &docker-ecr-cache-defaults
         cache-on:
+          - .npmrc
```

Comment:
Just playing this back for my benefit: we can exclude `package.json` from the cache key because a stale lockfile will fail on the subsequent `pnpm install --offline`?

Reply:
Yeah. So our dev deps now only contains a .pnpm-store, it should remain valid if we change lets say the `description` field in our package.json.